chart/Chart.lock

```diff
@@ -1,5 +1,6 @@
 dependencies:
 - name: postgresql
-  version: 10.5.3
-digest: sha256:7104938113e866364a96f78802697be6bd121526cccbd06cae6a38827b7b36f9
-generated: "2022-06-11T22:19:11.389661+02:00"
+  repository: https://charts.bitnami.com/bitnami
+  version: 12.0.0
+digest: sha256:7de3ef06bed45277fef19fe8290f08d0b744c12aeb34eee9babf1e9d1a429cb2
```

Comment:
I see, thanks for the explanation. I was wondering why the whole .tgz was included, but that makes sense.

> However - if I understand correctly - this one is purely to make it easier in the future if we want to upgrade (or it makes it easy for users to upgrade if they want to them themselves? This does not introduce "necessity" of running helm update correct?

Yes, that's correct. Since the .tgz is still included, people shouldn't need to run `helm dependency update` just to install the chart. This should just mean that all it will take to update the version of the bitnami/postgresql chart used here is to update the version in the `Chart.yaml` and then run `helm dependency update` and it should pull in the new chart .tgz and update the `Chart.lock`.

Reply:
Perfect! Thanks for doing it this way :)